From dcaa55543b159f3cb88062ea3a8a35a62bb0a2c9 Mon Sep 17 00:00:00 2001
From: Juan Gutierrez <juan.gutierrez@nxp.com>
Date: Wed, 1 Jun 2016 21:16:06 -0500
Subject: [PATCH 09/11] mx6sxscm: set the correct gpio for LCD backlight

To enable the LCD splashcreen on uboot when selecting
the panel MCIMX28LCD the backlight needs to be enabled.

This patch fix the selection of the right gpio  on the
board file

Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
---
 board/freescale/mx6sxscm/mx6sxscm.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/board/freescale/mx6sxscm/mx6sxscm.c b/board/freescale/mx6sxscm/mx6sxscm.c
index 7444636..49aa456 100644
--- a/board/freescale/mx6sxscm/mx6sxscm.c
+++ b/board/freescale/mx6sxscm/mx6sxscm.c
@@ -383,7 +383,8 @@ void do_enable_parallel_lcd(struct lcd_panel_info_t const *dev)
 	gpio_direction_output(IMX_GPIO_NR(3, 27) , 1);
 
 	/* Set Brightness to high */
-	gpio_direction_output(IMX_GPIO_NR(4, 18) , 1);
+	gpio_direction_output(IMX_GPIO_NR(4, 10) , 1);
+
 }
 
 static struct lcd_panel_info_t const displays[] = {{
-- 
1.9.1

